kalite/topic_tools/__init__.py

```diff
@@ -67,6 +78,20 @@ def get_topic_tree(force=False, annotate=False, channel=None, language=None, par
         TOPICS = {}
     if TOPICS.get(channel) is None:
         TOPICS[channel] = {}
+
+    if settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP and not force:
```

Comment:
>  Moving around that block of code had nothing to do with that,

Moving the code block was about only hitting `softload_json` once. I wasn't just cleaning up code, I was fixing it by moving it. Since the original intention was to avoid stuff that made the RPi way too slow, I think parsing the topic tree json once less is okay to push for in the scope of the PR.

I see that it might have delivered results that didn't respect the `annotated` kwarg, thanks for fixing that.. I don't know if it would have mattered since these functions are really bloated kwarg-wise :)